modular_skyrat/modules/modular_items/lewd_items/code/lewd_chemistry/reagents/incubus_draft.dm

```diff
@@ -135,26 +136,32 @@
 	return ..()
 
 /datum/reagent/drug/aphrodisiac/incubus_draft/overdose_effects(mob/living/carbon/human/exposed_mob)
+	// Check if overdosing on incubus draft and succubus milk simultaneously, to prevent chat spam
+	var double_dosing = FALSE
+	for(var/r in exposed_mob.reagents.reagent_list)
+		var/datum/reagent/reagent = r
+		if(reagent.name == "succubus milk" && reagent.overdosed)
```

Comment:
my previous comment would remove this section, but please *never* check for reagents like this

Reply:
In my defense, it's done like this all over the place in the code, and isn't as bad as you think. Reagent lists are never that long, it may even be a less costly operation than locate(). But I agree locate is much neater looking. Anyway fixed!